cmd/medius/docs/docs.go

```diff
@@ -60,10 +60,12 @@ func run(options *common.Options) error {
 		}
 
 		focusMatched = true
-		log := common.Logger(p.Artifact)
-		name := p.Artifact.Metadata().Name
 
-		description, err := createDescription(p.Artifact, options.PublishDocsOptions.Registry)
+		artifact := p.Artifacts[0]
+		log := common.Logger(artifact)
+		name := artifact.Metadata().Name
+
+		description, err := createDescription(artifact, options.PublishDocsOptions.Registry)
```

Comment:
We should add the supported architectures into the description. Maybe in a follow up.

Reply:
Please create an issue for that.

Reply:
Done: https://github.com/kubevirt/containerdisks/issues/83